data/migrations/V0102__drop_ofec_rad_mv.sql

```diff
@@ -0,0 +1,56 @@
+/*
+
+Addresses issue #3333: Add assignment_update to /rad-analyst/
+
+--Drop function `fix_party_spelling`
+- Replace view `rad_cmte_analyst_search_vw`
+    - Need to drop it because we're adding columns (idx, assignment_update)
+    - Update column names to match `ofec_rad_mv`
+- Drop materialized view `ofec_rad_mv`
+
+*/
+
+SET search_path = public;
+
+DROP MATERIALIZED VIEW ofec_rad_mv;
+
+DROP FUNCTION IF EXISTS fix_party_spelling(branch text);
+
+----
+
+SET search_path = disclosure, pg_catalog;
+
+DROP VIEW rad_cmte_analyst_search_vw;
+
+CREATE VIEW rad_cmte_analyst_search_vw AS
+    SELECT row_number() OVER () AS idx,
+        ra.cmte_id AS committee_id,
+        cv.cmte_nm AS committee_name,
+        an.anlyst_id AS analyst_id,
+        (an.valid_id::numeric) AS analyst_short_id,
+        CASE
+            WHEN an.branch_id = 1 THEN 'Authorized'
+            WHEN an.branch_id = 2 THEN 'Party/Non Party'
+            ELSE NULL::text
+        END AS rad_branch,
+        an.firstname AS first_name,
+        an.lastname AS last_name,
+        to_tsvector((((an.firstname)::text || ' '::text) || (an.lastname)::text)) AS name_txt,
+        an.telephone_ext,
+        t.anlyst_title_desc AS analyst_title,
+        an.email AS analyst_email,
+        ra.last_rp_change_dt AS assignment_update
+    FROM rad_pri_user.rad_anlyst an,
+        rad_pri_user.rad_assgn ra,
+        cmte_valid_fec_yr cv,
+        rad_pri_user.rad_lkp_anlyst_title t
+    WHERE an.status_id = 1
+        AND an.anlyst_id <> 999
+        AND cv.fec_election_yr = date_part('year', now())::integer + (date_part('year', now())::integer % 2)
+        AND an.anlyst_id = ra.anlyst_id
```

Comment:
Joins using JOIN syntax are preferred to those with a FROM list and WHERE conditions, for clarity.